client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -138,24 +266,21 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 								disabled={ isLoading }
 								onClick={ () => {
 									wcpayTracks.recordEvent(
-										wcpayTracks.events
-											.DISPUTE_ACCEPT_MODAL_VIEW,
+										disputeAcceptAction.acceptButtonTracksEvent,
```

Comment:
Having to parameterise the button names and the tracks events is a signal that the UI is too generic. It's ok to implement a simple component with hard-coded content and single purpose.